community/index.md

```diff
@@ -8,6 +8,7 @@ title: Get in touch!
     <p>We'd love to have your feedback, suggestions, and will be happy to help with anything you need. Just get in touch!</p>
 
     <ul>
+      <li><a href="https://lists.apache.org/list.html?jclouds.apache.org">Browse and participate</a> in our mailing lists</li>
```

Comment:
I don't have a strong opinion neither (happy to change to `user@`), but let me explain the purpose of this change: encourage participation.

PonyMail allows users to subscribe to specific threads, or even to participate in the discussions without having to subscribe to the lists, which can be "difficult" or inconvenient for users that are just interested in a particular thing. If we want to encourage participation from newcomers, perhaps is better to direct them by default to an active list instead to a very quiet one, so they see there is actually people in there. With the current email rate we could assume emails coming to the dev list by mistake.

I'm happy to default to the user list though, as it makes more sense and it could be the *right list*. However, I think it could be more convenient to default to dev. Dunno! :)